app/models/miq_template.rb

```diff
@@ -30,4 +30,8 @@ def self.eligible_for_provisioning
   end
 
   def active?; false; end
+
+  def supports_provisioning?
+    !ems_id.nil?
```

Comment:
@lfu, currently method that build Templates available for Provisioning uses same logic, see here https://github.com/ManageIQ/manageiq/blob/master/app/models/miq_template.rb#L29